src/main/controllers/YourResponseFormController.ts

```diff
@@ -16,12 +17,34 @@ import {
 } from '../helpers/controller/CheckYourAnswersET3Helper';
 import { getFlagValue } from '../modules/featureFlag/launchDarkly';
 import CollectionUtils from '../utils/CollectionUtils';
+import DocumentUtils from '../utils/DocumentUtils';
+import ObjectUtils from '../utils/ObjectUtils';
+import RespondentUtils from '../utils/RespondentUtils';
+import StringUtils from '../utils/StringUtils';
 
 export default class YourResponseFormController {
   constructor() {}
   public get = async (req: AppRequest, res: Response): Promise<void> => {
-    const redirectUrl = setUrlLanguage(req, PageUrls.YOUR_RESPONSE_FORM);
+    const redirectUrl: string = setUrlLanguage(req, PageUrls.YOUR_RESPONSE_FORM);
     const welshEnabled = await getFlagValue(TranslationKeys.WELSH_ENABLED, null);
+    const languageParam: string = getLanguageParam(req.url);
+    const selectedRespondent: RespondentET3Model = RespondentUtils.findSelectedRespondentByRequest(req);
+    let et3FormId: string = '';
```

Comment:
Suggest to put et3FormId function into a helper

Reply:
Added new method to DocumentUtils as findET3FormIdByRequest.